examples/SSVEP/plot_classify_ssvep_mdm.py

```diff
@@ -0,0 +1,239 @@
+raw_ext.plot(duration=n_seconds, start=14, n_channels=24,
+             scalings={'eeg': 5e-4}, color={'eeg': 'steelblue'})
+
+###############################################################################
+# As it can be seen on this example, the subject is watching the 13Hz
+# stimulation and the EEG activity is showing an increase activity in this
+# frequency band while other frequency have a lower amplitude.
+#
+# Spatial covariance for SSVEP
+# ----------------------------
+#
+# The covariance matrices will be estimated using the Ledoit-Wolf shrinkage
+# estimator on the extended signal.
+
+cov_ext_trials = Covariances(estimator='lwf').transform(ext_trials)
+
+# This plot shows an example of a covariance matrix observed for each class:
+
+plt.figure(figsize=(7, 7))
+for i, l in enumerate(event_id):
+    ax = plt.subplot(2, 2, i+1)
+    plt.imshow(cov_ext_trials[events[:, 2] == event_id[l]][0],
+               cmap=plt.get_cmap('RdBu_r'),
+               interpolation='nearest')
+    plt.title('Cov for class: '+l)
+    plt.xticks([])
+    if i == 0 or i == 2:
+        plt.yticks(np.arange(len(info['ch_names'])),info['ch_names'])
+        ax.tick_params(axis='both', which='major', labelsize=7)
+    else:
+        plt.yticks([])
+
+###############################################################################
+# It appears clearly that each class yields a different structure of the
+# covariance matrix. Each stimulation (13, 17 and 21 Hz) generating higher
+# covariance values for EEG signal filtered at the proper bandwith and no
+# activation at all for the other bandwith. The resting state, where the
+# subject focus on the center of the display and far from all blinking
+# stimulus, shows an activity with higher correlation in the 13Hz frequency
+# and lower but still visible activity in the other bandwith.
+#
+# Classify with MDM
+# -----------------
+# Plotting mean of each class
+
+cov_centers = np.empty((len(event_id), 24, 24))
+for i, l in enumerate(event_id):
+    cov_centers[i] = mean_riemann(cov_ext_trials[events[:, 2] == event_id[l]])
+
+plt.figure(figsize=(7, 7))
+for i, l in enumerate(event_id):
+    ax = plt.subplot(2, 2, i+1)
+    plt.imshow(cov_centers[i],
+               cmap=plt.get_cmap('RdBu_r'),
+               interpolation='nearest')
+    plt.title('Cov mean for class: '+l)
+    plt.xticks([])
+    if i == 0 or i == 2:
+        plt.yticks(np.arange(len(info['ch_names'])),info['ch_names'])
+        ax.tick_params(axis='both', which='major', labelsize=7)
+    else:
+        plt.yticks([])
+
+###############################################################################
+# Minimum distance to mean is a simple and robust algorithm for BCI decoding
+
+cv = RepeatedKFold(n_splits=2, n_repeats=10, random_state=42)
+mdm = MDM(metric=dict(mean='riemann', distance='riemann'))
+scores = cross_val_score(mdm, cov_ext_trials, events[:, 2], cv=cv, n_jobs=1)
+print("MDM accuracy: {:.2f}% +/- {:.2f}".format(np.mean(scores)*100, 
```

Comment:
I suggest you leave a comment with the expected output of this final accuracy... just as a reference on what it should have given if everything went well

Reply:
Yes, good idea.